### PR TITLE
Update Kops dependency for Azure Blob Storage support

### PR DESCRIPTION
**Pull request by rifelpet**
This updates the kops dependency to the current master HEAD. note that the go.mod replace directive is changing from `v1.19.0-beta.1` to `v1.19.0-alpha.5.0.20201129060248-dc1c2210cf77` because `v1.19.0-alpha.5` is the most recent tag cut from the master branch before we created the release-1.19 branch.

This also updates klog to v2.

### CODE REVIEW

**Comment by justinsb** [comment]
So I think I messed this up with the tools/deb-tools subdirectory.

The next 3 PRs should fix it properly, but if you want this one to pass tests I think this should do it...


```
pushd tools/deb-tools
go mod vendor
bazel run //:gazelle
popd
```

**Comment by rifelpet** [comment]
Thanks @justinsb, updated. I think one issue is gazelle is using the wrong prefix for the deb-tools module: https://github.com/kopeio/etcd-manager/blob/e694496d2e2de6d73df764b46373bcdd4d7ce183/tools/deb-tools/BUILD.bazel#L8

**Comment by justinsb** [comment]
Thanks @rifelpet and you're right - that should be `deb-tools` not `deb-extract`.  It might work anyway, but I agree we should fix it :-)

File: tools/deb-tools/bazel-deb-tools [hunk 1]
```diff
@@ -0,0 +1 @@
+/private/var/tmp/_bazel_prifel/1b4f270d25cb3a9340b9e5aad3a4249e/execroot/__main__
```
Comment (justinsb) [inline tools/deb-tools/bazel-deb-tools:1]: This (symlink?) probably shouldn't be committed.. though it looks like it isn't causing problems...

Comment (rifelpet) [reply tools/deb-tools/bazel-deb-tools]: ah thanks, i'm not sure which command I ran that generated that symlink but it should probably get gitignored if its still created after deb-tools is fixed.

**Comment by justinsb** [comment]
Removing travis in #377, so not going to wait for it.  The same tests are passing in github actions.

/approve
/lgtm

**Pull request merged**